instrumentation/lettuce-5.0/src/main/java/io/lettuce/core/StatefulRedisConnectionImpl.java

```diff
@@ -0,0 +1,21 @@
+package io.lettuce.core;
+
+import java.time.Duration;
+
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.NewField;
+import com.newrelic.api.agent.weaver.Weave;
+
+import io.lettuce.core.api.StatefulConnection;
+import io.lettuce.core.codec.RedisCodec;
+
+@Weave(type = MatchType.BaseClass)
+public abstract class StatefulRedisConnectionImpl<K, V> implements StatefulConnection<K, V> {
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.